src/main/java/org/motechproject/mots/mapper/EnumsMapper.java

```diff
@@ -1,11 +1,13 @@
 package org.motechproject.mots.mapper;
 
+import org.mapstruct.Mapper;
 import org.motechproject.mots.domain.enums.EducationLevel;
 import org.motechproject.mots.domain.enums.FacilityType;
 import org.motechproject.mots.domain.enums.Gender;
 import org.motechproject.mots.domain.enums.Language;
 import org.motechproject.mots.domain.enums.Literacy;
 
+@Mapper(componentModel = "spring")
```

Comment:
did you checked if all other mappers using this one are working fine after this change?

Reply:
I have checked saving and editing CHW and Incharge. Everything seems to work correctly